azurerm/resource_arm_app_service.go

```diff
@@ -33,6 +33,16 @@ func resourceArmAppService() *schema.Resource {
 
 			"location": locationSchema(),
 
+			"kind": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
we should add a Default value for this for existing users, e.g. "app" (which appears to be the default for a Web App/Website/App Service)

Reply:
Sure. And where did your find those available values? My knowledge about "functionapp" is provided by the Azure Functions team.

Reply:
from memory creating an App Service / Function App / Mobile API in the portal and either seeing it in the output of resources.azure.com / API responses / the export template function - although they may also be in the Go SDK